src/components/ConnectModal/ConnectModal.js

```diff
@@ -0,0 +1,85 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { ButtonText, GU, Modal, textStyle, useTheme } from '@aragon/ui'
+import providers from '../../ethereum-providers'
+import ProviderCard from './ProviderCard'
+
+function ConnectModal({ visible, onConnect, onClose }) {
+  const theme = useTheme()
+
+  const modalWidth = useCallback(viewport => {
+    return 130 * GU
+  }, [])
+
+  return (
+    <Modal visible={visible} width={modalWidth} onClose={onClose}>
```

Comment:
Reminder: this part is not mobile friendly.

Reply:
Yes, it’s not done yet.